requirements.txt

```diff
@@ -0,0 +1 @@
+mysql-connector-python==8.0.25
```

Comment:
Getting this error while running (within Docker):

```
ModuleNotFoundError: No module named 'dotenv'
```

This file needs to include the [`python-dotenv` library](https://pypi.org/project/python-dotenv/).